glean-core/src/metrics/denominator.rs

```diff
@@ -81,15 +90,51 @@ impl DenominatorMetric {
     /// Gets the currently stored value as an integer.
     ///
     /// This doesn't clear the stored value.
-    pub fn test_get_value(&self, glean: &Glean, storage_name: &str) -> Option<i32> {
+    pub fn test_get_value(&self, ping_name: Option<String>) -> Option<i32> {
+        crate::block_on_dispatcher();
```

Comment:
Wait, uniffi brings the dispatcher to glean-core? Nice.

Reply:
Yes!